core/src/main/java/com/exadel/aem/toolkit/bundle/lists/datasource/ItemComponentDatasource.java

```diff
@@ -42,27 +42,27 @@
 import com.adobe.granite.ui.components.ds.ValueMapResource;
 
 /**
- * Servlet that implements {@code datasource} pattern for populating a TouchUI {@code select} widget
- * with item components that have acl-item-component group in a Granite datasource
+ * Servlet that implements {@code datasource} pattern for populating a TouchUI {@code Select} widget
+ * with item components that have aatListItem property in a Granite datasource
  */
 @Component(
     service = Servlet.class,
     property = {
-        "sling.servlet.resourceTypes=aem-custom-lists/datasources/list-items",
+        "sling.servlet.resourceTypes=/apps/authoring-toolkit/lists/datasources/list-items",
         "sling.servlet.methods=" + HttpConstants.METHOD_GET
     }
 )
 public class ItemComponentDatasource extends SlingSafeMethodsServlet {
     private static final Logger LOG = LoggerFactory.getLogger(ItemComponentDatasource.class);
 
-    private static final String SELECT_STATEMENT = "SELECT * FROM [cq:Component] AS s WHERE ISDESCENDANTNODE(s,'/apps') AND [componentGroup] = 'acl-item-component'";
+    private static final String SELECT_STATEMENT = "SELECT * FROM [cq:Component] AS s WHERE ISDESCENDANTNODE(s,'/apps') AND [aatListItem] = 'true'";
 
     private static final String PN_VALUE = "value";
     private static final String PN_TEXT = "text";
 
     /**
      * Processes {@code GET} requests to the current endpoint to add to the {@code SlingHttpServletRequest}
-     * a {@code datasource} object filled with item components that have acl-item-component group in a Granite datasource
+     * a {@code datasource} object filled with item components that are list items
```

Comment:
(cosmetic) I'd rather written `DataSource` in Camel Case everywhere

Reply:
Fixed